### DISCUSSION TITLE
LOW FPS

### DISCUSSION THREAD

**Discussion by Rockyaner** [Stellarium/stellarium#3325] [General]
Hello good morning. Could someone help me? I have high FPS drops when running 2 clients, in 1 of them the fps starts to drop after a while online and in the other everything is still fine. I already bought my computer and the problem is not in it. Does anyone know how to fix it?

**Comment by gzotti** [comment]
https://github.com/Stellarium/stellarium/wiki/FAQ#reporting-a-crash
